CHANGELOG.next.toml

```
@@ -11,6 +11,14 @@
 # meta = { "breaking" = false, "tada" = false, "bug" = false, "target" = "client | server | all"}
 # author = "rcoh"
 
+[[smithy-rs]]
+message = """
+Fix rust rendering of @error structs when fields have default implementation
+"""
+references = ["smithy-rs#3182"]
+meta = { "breaking" = false, "tada" = false, "bug" = true, "target" = "all"}
```

```suggestion
meta = { "breaking" = false, "tada" = false, "bug" = true, "target" = "client"}
```
This change only affects clients.